docs/testing/testing-with-django.md

```diff
@@ -5,7 +5,7 @@
 Zulip uses the Django framework for its Python backend. We
 use the testing framework from
 [django.test](https://docs.djangoproject.com/en/3.2/topics/testing/)
-to test our code. We have over a thousand automated tests that verify that
+to test our code. We have over three thousand automated tests that verify that
```

Comment:
How did you get your number? I see 4500 or so.

```
$ git grep 'def test_' | wc
   4543   22779  409281
```

I think best would be to just say "thousands of" regardless.

Reply:
During running `./tools/test-backend zerver.tests`. 

![zuliptests](https://github.com/zulip/zulip/assets/69990740/f10f12cb-a43e-4dc1-9cfe-89fae83b0a4a)

So, should I make it: "thousands of"?

Reply:
OK. Some tests don't run in the default mode -- you have to pass some extra options to get them all. And yes, "thousands of".

Reply:
Made the changes. It's now making sense. :)